Detectors/GRP/calibration/src/LHCIFfileReader.cxx

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "GRPCalibration/LHCIFfileReader.h"
+#include "TSystem.h"
+#include <fstream>
+
+namespace o2
+{
+namespace grp
+{
+
+void LHCIFfileReader::loadLHCIFfile(const char* fileName)
+{
+  // load the LHC IF file into a string
+  char* expandedFileName = gSystem->ExpandPathName(fileName);
+  std::ifstream ifs(expandedFileName);
```

Comment:
why do you need `gSystem`?  If you wand to check file existence or expand its path, you can use
`o2::utils::Str::pathExists(filename);`
and 
`o2::utils::Str::getFullPath(filename);`

Reply:
Ok, I did not know.